CHANGELOG.md

```diff
@@ -60,15 +77,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: The reverse air underwater in Training Chamber now has a method with and without Space Jump (Advanced and Expert respectively). This can be used to get to the bomb slot as well as the door to Fortress Transport Access.
 
-## [7.4.1] - 2024-03-13
+## [7.4.2] - 2024-03-13
 
-### AM2R
-- Fixed: Hitting Zetas with Charge Beam works again.
+This release is identical to 7.4.0 and was released to revert 7.4.1.
 
-### Cave Story
-- Fixed: Cave Story exports with CS:Tweaked now prioritize the mod-specific files over Freeware's. This solves several issues with missing graphics when exporting over a Freeeware game.
-- Fixed: Missing graphical assets for rando-exclusive inventory entries in Cave Story: Tweaked exports
+## [7.4.1] - 2024-03-13
 
+This release was abandoned as it caused issues.
```

Comment:
Wouldnt it be better to still list out what did change? AKA have duplication in both 7.4.1 and 7.5.0